pycbc/pnutils.py

```diff
@@ -652,11 +664,11 @@ def _dtdv_cutoff_velocity(m1, m2, chi1, chi2):
     dtdv0, dtdv2, dtdv3, dtdv4, dtdv5, dtdv6, dtdv6log, dtdv7 = _dtdv_coeffs(m1, m2, chi1, chi2)
 
     def dtdv_func(v):
-        return 1. + v * v * (dtdv2 + v * (dtdv3 \
-                + v * (dtdv4
-                + v * (dtdv5
-                + v * ((dtdv6 + dtdv6log*3.*log(v))
-                + v * dtdv7)))))
+        return 1. + v * v * (dtdv2 + v * (dtdv3 +
+                             v * (dtdv4 +
+                             v * (dtdv5 +
+                             v * ((dtdv6 + dtdv6log*3.*numpy.log(v)) +
+                             v * dtdv7)))))
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->